[2025-07-09T08:23:46.436+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-09T08:23:46.481+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ETL_airflare_data.extract manual__2025-07-09T08:23:44.067884+00:00 map_index=9 [queued]>
[2025-07-09T08:23:46.499+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ETL_airflare_data.extract manual__2025-07-09T08:23:44.067884+00:00 map_index=9 [queued]>
[2025-07-09T08:23:46.500+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 3
[2025-07-09T08:23:46.543+0000] {taskinstance.py:2330} INFO - Executing <Mapped(_PythonDecoratedOperator): extract> on 2025-07-09 08:23:44.067884+00:00
[2025-07-09T08:23:46.567+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=7806) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-07-09T08:23:46.571+0000] {standard_task_runner.py:63} INFO - Started process 7839 to run task
[2025-07-09T08:23:46.571+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'ETL_airflare_data', 'extract', 'manual__2025-07-09T08:23:44.067884+00:00', '--job-id', '686', '--raw', '--subdir', 'DAGS_FOLDER/Untitled-1.py', '--cfg-path', '/tmp/tmpr9ri_w96', '--map-index', '9']
[2025-07-09T08:23:46.574+0000] {standard_task_runner.py:91} INFO - Job 686: Subtask extract
[2025-07-09T08:23:46.709+0000] {task_command.py:426} INFO - Running <TaskInstance: ETL_airflare_data.extract manual__2025-07-09T08:23:44.067884+00:00 map_index=9 [running]> on host e6a32b72ede0
[2025-07-09T08:23:46.882+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ETL_airflare_data' AIRFLOW_CTX_TASK_ID='extract' AIRFLOW_CTX_EXECUTION_DATE='2025-07-09T08:23:44.067884+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-09T08:23:44.067884+00:00'
[2025-07-09T08:23:46.885+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-09T08:23:46.886+0000] {Untitled-1.py:137} INFO - Начало извлечения данных для направления VVO-IKT
[2025-07-09T08:23:47.398+0000] {Untitled-1.py:62} INFO - Создание подключения к S3
[2025-07-09T08:23:47.398+0000] {Untitled-1.py:65} INFO - Подключение установлено
[2025-07-09T08:23:47.413+0000] {base.py:84} INFO - Using connection ID 'minio' for task execution.
[2025-07-09T08:23:47.414+0000] {connection_wrapper.py:384} INFO - AWS Connection (conn_id='minio', conn_type='aws') credentials retrieved from login and password.
[2025-07-09T08:23:48.019+0000] {Untitled-1.py:71} INFO - Загрузка данных в S3
[2025-07-09T08:23:48.573+0000] {Untitled-1.py:82} INFO - Загрузка завершена успешно
[2025-07-09T08:23:48.574+0000] {Untitled-1.py:150} INFO - Данные успешно сохранены во временное хранилище
[2025-07-09T08:23:48.575+0000] {Untitled-1.py:151} INFO - Данные успешно извлечены и переданы в Xcom
[2025-07-09T08:23:48.577+0000] {python.py:237} INFO - Done. Returned value was: {'path': {'bucket': 'airflare', 'key': 'temp/response_VVO_IKT.json'}, 'rout': {'origin': 'VVO', 'destination': 'IKT', 'bd': 'Владивосток_Иркутск_return'}}
[2025-07-09T08:23:48.578+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-09T08:23:48.606+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=ETL_airflare_data, task_id=extract, run_id=manual__2025-07-09T08:23:44.067884+00:00, map_index=9, execution_date=20250709T082344, start_date=20250709T082346, end_date=20250709T082348
[2025-07-09T08:23:48.685+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-07-09T08:23:48.697+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
